src/packages/ddoc/components/bottom-toolbar.tsx

```diff
@@ -204,18 +220,30 @@ const BottomToolbar = ({
                     <button
                       ref={textFormattingButtonRef}
                       onClick={() => tool.onClick()}
+                      className={cn(
+                        'h-9 w-9 flex justify-center items-center rounded',
+                        {
```

Comment:
Let's use IconButtons for these, with isActive props and ghost variant

Reply:
Need `ref` prop in the IconButton component to use it

Reply:
Done